pages/style-guide/html.md

```diff
@@ -4,74 +4,170 @@
 
 This page outlines the style guide for HTML pages in all jQuery projects. These rules apply to web sites, demo pages, inline examples, test pages, etc. Exceptions are allowed for pages that must violate the rules by their very nature, e.g., a page that tests XHTML.
 
-## Doctype
-Always use the minimal, versionless doctype.
+## Linting
 
-```
-<!doctype html>
-```
+Use [grunt-html](https://www.npmjs.com/package/grunt-html) to detect errors and potential problems. Most jQuery projects have a Grunt task for linting all HTML files: `grunt htmllint`.
```

Comment:
I don't think Chassis has this (although I could be missing it).  Might be useful to add that validation, especially with the documentation/style guides in Chassis.